sdk.go

```diff
@@ -0,0 +1,11 @@
+package xcaddy
+
+import "os"
+
+func GetGo() string {
+	sdk := os.Getenv("XCADDY_SDK")

```

Comment:
```suggestion
	sdk := os.Getenv("XCADDY_GO")
```

What about this?

Reply:
I was thinking `XCADDY_GO_CMD` (spoke with @mohammed90 on slack and we agreed, I guess we forgot to write it here though)

Reply:
Sure, or maybe `XCADDY_WHICH_GO` (like the shell command, `which go`) -- either way though. I don't have a strong preference; just not SDK if possible. :)

Reply:
Will fix that